### PR TITLE
Laos EOC proof of concept dashboard

### PR DESCRIPTION
**Pull request by IgorNadj**
### Changes:

- Add network config to be able to connect to Laos DHIS
- Add code to id mappings needed for this dashboard
- Create new data builder
- Add dashboard

---

### .env config needed:
```
# staging server
LAO-PEOPLES-DEMOCRATIC-REPUBLIC_DHIS_CLIENT_ID=
LAO-PEOPLES-DEMOCRATIC-REPUBLIC_DHIS_CLIENT_SECRET=
LAO-PEOPLES-DEMOCRATIC-REPUBLIC_DHIS_USERNAME=
LAO-PEOPLES-DEMOCRATIC-REPUBLIC_DHIS_PASSWORD=
LAO-PEOPLES-DEMOCRATIC-REPUBLIC_DHIS_API_URL=https://dhis2.world/laohmis
LAO-PEOPLES-DEMOCRATIC-REPUBLIC_DHIS_AUTH_TOKEN_HOST=https://dhis2.world/
LAO-PEOPLES-DEMOCRATIC-REPUBLIC_DHIS_AUTH_TOKEN_PATH=/laohmis/uaa/oauth/token
```
// fill out missing from lastpass

### CODE REVIEW

File: packages/web-config-server/src/apiV1/dataBuilders/generic/count/countEventsPerPeriodByDataValue2.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import moment from 'moment';
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import { DataPerPeriodBuilder } from 'apiV1/dataBuilders/DataPerPeriodBuilder';
+import { groupEventsByPeriod } from '@tupaia/dhis-api';
+import { getAllDataElementCodes } from '../../helpers';
+
+/**
+ * @abstract
+ */
+class DataByValueBuilder2 extends DataBuilder {
```
Comment (IgorNadj) [inline packages/web-config-server/src/apiV1/dataBuilders/generic/count/countEventsPerPeriodByDataValue2.js:17]: This is bad but intentional. Cleanup split off to another card to make this one easier to review. Cleanup card: #2203

**Review by nick-beyondessential — APPROVED** [review]
Ok a bit nervous about approving this PR because of code duplication but realise it is into a epic branch for expediency of demo and have been assured that https://github.com/beyondessential/tupaia-backlog/issues/2203 will be implemented before merging into dev.

**Comment by edmofro** [comment]
@IgorNadj did this one go on the reviews channel? Good practice to post them there for transparency, even if you know who will be reviewing it (you can even add the caught emoji yourself if you've had side conversations and know they're on it)

**Comment by edmofro** [comment]
Oh apologies - it totally did! I came to it from your other link and hadn't noticed it, but that's just me not paying attention. Sorry!

**Pull request merged**